compiler/luci/pass/src/PropagateQParamBackwardPass.test.cpp

```diff
@@ -165,3 +278,33 @@ TEST(PropagateQParamBackwardPassTest, subsequent_propagation)
   EXPECT_EQ(3.0, graph.input->quantparam()->scale[0]);
   EXPECT_EQ(3, graph.input->quantparam()->zerop[0]);
 }
+
+TEST(PropagateQParamBackwardPassTest, reshape)
```

Comment:
Added one positive/negative test for reshape. I didn't add tests for transpose, because qparam propagation works in the exactly same way for both Ops.